<launch>
    <arg name="outfile" default="/dev/null" />

    <node pkg="ut_multirobot_sim" type="simulator_link" name="simulator" cwd="node"
        args="-sim_config $(find ut_multirobot_sim)/config/gdc_gym_gen/sim_config.lua -scene_config $(find ut_multirobot_sim)/config/gdc_gym_gen/scene.xml -speedup_factor 1.0 --localize --use_pedsim" />

    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ut_multirobot_sim)/visualization.rviz" /> -->

    <node pkg="graph_navigation" type="navigation" name="graph_navigation"
        args="-map $(find ut_multirobot_sim)/maps/GDC1/GDC1.navigation.txt" />

    <include file="$(find ut_multirobot_sim)/config/gdc_gym_gen/pedsim_launch.launch" />

    <param name="enable_statistics" value="true" />
</launch>
